docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1220,7 +1220,77 @@ _____________________
 
-In order for PacketFence to allow Eduroam authentication, you need to configure an exclusive Eduroam source in Configuration->Authentication Sources and restart the RADIUS and IPTables services.
+Understanding of the eduroam authentication workflow.
+
+Local authentication
+^^^^^^^^^^^^^^^^^^^^
+
+image::internal-eduroam-authentication.png[scaledwidth="100%",alt="Internal eduroam authentication"]
+
+1: The device connects on the eduroam SSID.
+
+2: The access point forward the authentication request to the wireless controller.
+
+3: The controller sends the radius authentication to PacketFence on port 11812.
+
+4: PacketFence checks if it's a local REALM.
+
+5: If it's local REALM, PacketFence does a NTLM request to the AD to verify the identity.
+
+5a: The AD validated the credentials.
+
+5b: The AD did not validate the credentials. PacketFence sends a Radius Reject 
+
+5c: After a successful NTLM authentication, PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that mac address.
+
+6: If it's a not local REALM, PacketFence proxy the radius request to the eduroam servers
+
+7: The eduroam servers validate the identity
+
+8: PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that mac address.
+
+Configure the Eduroam source
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Go to the PacketFence administration web interface and go to *Configuration -> Policies and Access Control -> Authentication Sources*.
+
+Local *Exclusive Sources* and click on 'New excluse source' then 'Eduroam'.
+
+The information to setup that source could be found on the eduroam platform.
+
+Create the Connection Profile for local authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Go to *Configuration -> Policies and Access Control -> Connection Profiles -> New Connection Profile*
+
+Create the Connection Profile named 'Local and external eduroam authentication' Check "Automatically register devices" then create a SSID filter 'eduroam'. Make sure to add the Active Directory source to match on the local users.
+
+
+Inbound authentication (TLRS to PF)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+image::inbound-eduroam-authentication.png[scaledwidth="100%",alt="Inbound eduroam authentication"]
+
+1: Eduroam sends the radius authentication to a public IP address (NAT/PAT) bound to PacketFence on the management IP address (Management VIP for a cluster) on port 1812.
+
+2: PacketFence forwards the NTLM request to the Active Directory.
+
+3a: User identity successfuly authentication on the AD.
+
+3b: NTLM request fails because of a bad identity
+
+4: PacketFence replies to the Eduroam servers either a RADIUS Access Accept for a sucessful authentication or a RADIUS Access Reject for an unsuccessful authentication. PacketFence sets the REALM to eduroam for any sucessful authentication.
```

Comment:
eduroam.... RADIUS access-accept .... RADIUS access-reject... for all successful authentications.